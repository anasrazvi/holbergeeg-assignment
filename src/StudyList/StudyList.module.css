.innerContainer{
    border: 1px solid #ccc;
    margin: 10px;
    height: 96%;
    width: calc(100% - (350px));
    float: left;
}