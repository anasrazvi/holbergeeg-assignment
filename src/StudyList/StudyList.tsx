import {SectionHeader} from '../SectionHeader/SectionHeader';
import {Grid} from '../Grid/Grid';
import {myStore} from '../App'
import RightContainerCSS from './StudyList.module.css';

export const StudyList: React.FC = () => {
    return (<div className={RightContainerCSS.innerContainer}>
                <SectionHeader label="Study List:"></SectionHeader>
                <Grid store={myStore}></Grid>
          </div>)
  }