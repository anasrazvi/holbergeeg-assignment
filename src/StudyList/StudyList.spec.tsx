import renderer from 'react-test-renderer';
import {StudyList} from '../StudyList/StudyList';

describe("Study List Component", () => {
    test('renders study list', () => {
      const studyList = renderer.create(<StudyList></StudyList>).toJSON();
      expect(studyList).toMatchSnapshot();
    });
});