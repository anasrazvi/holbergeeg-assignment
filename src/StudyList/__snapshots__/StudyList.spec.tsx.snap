// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Study List Component renders study list 1`] = `
<div
  className="innerContainer"
>
  <div
    className="sectionHeader"
  >
    Study List:
  </div>
  <div
    className="gridContainer"
  >
    <div
      className="header"
    />
    <div />
  </div>
</div>
`;
